Modules/Scripted/DICOMLib/DICOMUtils.py

```diff
@@ -745,7 +769,7 @@ def getLoadablesFromFileLists(fileLists, pluginClassNames=None, messages=None, p
         return loadablesByPlugin, loadEnabled
 
     if pluginClassNames is None:
-        pluginClassNames = list(slicer.modules.dicomPlugins.keys())
+        pluginClassNames = getEnabledPlugins()
```

Comment:
Wouldn't this break backward compatibility?
For example, previous calls to `DICOMUtils.loadSeriesByUID` will now exclude disabled plugins unless `pluginClassNames` are already provided when calling.

I suppose anyone using the function now has to update to
```py
DICOMUtils.loadSeriesByUID(seriesUIDs, pluginClassNames=list(slicer.modules.dicomPlugins.keys())
```

to restore the previous behavior.

Reply:
This was a bug (it was not specified in the documentation that even disabled plugins will be used). If someone wants to use all plugins then it is possible to specify a custom list in the newly added parameter.

Reply:
Thanks. That makes sense.